Diagnostic/diagnostic.py

```diff
@@ -879,13 +905,19 @@ def install_omi():
             hutil.error("OMI install failed " + str(maxTries) + " times. Giving up...")
             return 1, "OMI install failed " + str(maxTries) + " times"
 
+    shouldRestartOmi = False
+
+    # Check if OMI is configured to listen to 1270 and reconfigure if so.
+    omiListensTo1270 = RunGetOutput("grep '^httpsport=0,1270' /etc/opt/omi/conf/omiserver.conf")[0] is 0
```

Comment:
Thanks. I was worried about those possibilities as well. Will strengthen the checking logic and push an update.